credsweeper/common/constants.py

```diff
@@ -17,8 +17,10 @@ class KeywordPattern:
     # Authentication scheme ( oauth | basic | bearer | apikey ) precedes to credential
     value = r"(?P<value_leftquote>((b|r|br|rb|u|f|rf|fr|\\{0,8})?[`'\"]){1,4})?" \
             r"( ?(oauth|bot|basic|bearer|apikey|accesskey) )?" \
-            r"(?P<value>(?:\{[^}]{3,8000}\})|(?:<[^>]{3,8000}>)|" \
-            r"(?(value_leftquote)(?:\\[tnrux0-7][0-9a-f]*|[^`'\"\\])|(?:\\n|\\r|\\?[^\s`'\"\\,;])){3,8000})" \
+            r"(?P<value>" \
+            r"(?(value_leftquote)(?:\\[tnrux0-7][0-9a-f]*|[^`'\"\\])|(?:\\n|\\r|\\?[^\s`'\"\\,;])){3,8000}" \
+            r"|(?:\{[^}]{3,8000}\})|(?:<[^>]{3,8000}>)" \
+            r")" \
```

Comment:
Could you explain why those lines are switched?

Reply:
that regex collects full quoted value first "``[2ns8BH]ds$%^``"
previous - collects only in brackets "``[2ns8BH]``ds$%^"

Reply:
Hmm... why do you think bracket included value is correct value?
I think previous regex was intended to filter the bracket included values.

Reply:
https://github.com/Samsung/CredSweeper/actions/runs/9839488594/job/27161670040
- the test with random generation

Reply:
Okay i see.